Makefile

```diff
@@ -10,7 +10,7 @@ lint:
 	flake8
 
 test: all
-	QIIMETEST= nosetests
+	QIIMETEST= pytest
```

Comment:
TODO:

Update busywork before this PR is merged:

https://github.com/qiime2/busywork/blob/162dd4ecda77c09b3ad6a6fffcfae8405c1a0158/ci/master/variables.yaml#L22

Reply:
@andrewsanchez - please cut a PR on qiime2/busywork that addresses the change requested above - thanks!

Reply:
Looks like this is outdated.  Is there still something that needs to be done?  Should I remove  Here are all references to `nosetests`:

![image](https://user-images.githubusercontent.com/11708906/85171417-d37cea80-b223-11ea-9d9d-a7bf364c5388.png)

Should I remove nose from the q2cli runners?

Reply:
https://github.com/qiime2/busywork/pull/102